.github/workflows/runtests.yml

```diff
@@ -1,34 +1,37 @@
 name: Run tests
 
-on: [push, pull_request]
+on:
+   push:
+     branches:
+       - master
+   pull_request:
+     branches:
+       - master
 
 jobs:
-  test-julia-stable:
+  test:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ['1.0', '1.3', '1.4']
+        julia-version: ['1.3', '1.4', 'nightly']
         os: [ubuntu-latest, macOS-latest]
-      fail-fast: true
+        xcode: [ '11.4', '11.3.1' ]
+        cxxwrap: [ '0.9', '0.10' ]
+        exclude:
+        # only julia 1.4 on macOS
+        # choose one arbitrary xcode version for linux
```

Comment:
huh? xcode is macos specific, what does "choosing an xcode version for linux" even mean?

Reply:
the environment variable will be ignored but I cannot expand just one entry of an axis in a new dimension for the xcode version (at least I dont know of a quick way to do this in github yaml)